src/Swashbuckle.SwaggerGen/Annotations/SwaggerProducesResponseTypeAttribute.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Swashbuckle.SwaggerGen.Annotations
+{
```

Comment:
Before I remove them... XML comments are where IntelliSense sources its doc. While it adds to the code, consumers aren't reading the code at this level.

Are you sure you want them gone?